src/index.html

```diff
@@ -1,14 +1,16 @@
 <!DOCTYPE html>
-<html class="no-js"> 
-    <head>
-        <meta charset="utf-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge">
-        <title>Authors Haven Jawans</title>
-        <meta name="description" content="">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-    </head>
-    <body>
-        <section id="root"></section>
-    </body>
-</html>
+<html class="no-js">
 
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Authors Haven Jawans</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+
+<body>
+    <section id="app"></section>
+</body>
+
+</html>
```

Comment:
Please add a line at the end of all the files. It's easier if you download the Prettier extension.